src/components/ProjectCard/index.js

```diff
@@ -0,0 +1,200 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+import { useTheme } from '@material-ui/styles';
+
+import githubColorDictionary from '../data/gh-colors.json';
+import useStyles from './styles.js';
+
+const renderLanguageChip = (language, classes) => {
+  if (language === null) {
+    return '';
+  }
+  return (
+    <Typography className={classes.tagChipContainerStyle}>
+      <span
+        className={classes.languageLogoStyle}
+        style={{
+          backgroundColor: githubColorDictionary[language],
+        }}
+      />
+      <span className={classes.tagChipTextStyle}>{language}</span>
+    </Typography>
+  );
+};
+
+const renderTopicTags = (topics, classes) => {
+  if (topics === null) {
+    return '';
+  }
+  return topics.map((topic) => {
+    return (
+      <Typography key={topic} className={classes.tagChipContainerAltStyle}>
+        <span className={classes.tagChipTextStyle}>{topic}</span>
+      </Typography>
+    );
+  });
+};
+
+const renderAffiliationsTopicsTags = (topics, classes) => {
+  if (topics === null) {
+    return '';
+  }
+  return topics.map((topic) => {
+    return (
+      <Typography key={topic} className={classes.tagChipContainerStyle}>
+        <img
+          alt='affiliationTag'
+          className={classes.affiliationTopicLogoStyle}
+          src='/images/github-topic-tag-logo.png'
+        />
+        <span className={classes.tagChipTextStyle}>{topic}</span>
+      </Typography>
+    );
+  });
+};
+
+const renderGithubInfoComp = (classes, props, iconType) => {
+  let count;
+  let imageSrc;
+  if (iconType === 'Watch') {
+    count = props.watchers;
+    imageSrc = '/images/GithubEye.jpg';
+  } else {
+    count = props.stargazers;
+    imageSrc = '/images/GithubStar.jpg';
+  }
+  return (
+    <Typography className={classes.githubInfoStyle}>
+      <img
+        alt='githubInfoTag'
+        className={classes.githubInfoTopicLogoStyle}
+        src={imageSrc}
+      />
+      <span className={classes.tagChipTextStyle}>{iconType}</span>
+      <span className={classes.githubInfoIconSeparatorStyle}></span>
+      <span className={classes.tagChipTextStyle}>{count}</span>
+    </Typography>
+  );
+};
+
+const renderGithubInfoTags = (classes, props) => {
+  return (
+    <Grid className={classes.githubIconContainerStyle}>
+      {renderGithubInfoComp(classes, props, 'Watch')}
+      {renderGithubInfoComp(classes, props, 'Star')}
+    </Grid>
+  );
+};
+/**
+ *
+ * @param {*} props
+ */
+export default function ProjectCard(props) {
```

Comment:
I'm seeing a line limit error in eslint.  Is there any way some blank lines/comments can be removed to get it under the limit?

![image](https://user-images.githubusercontent.com/1535252/128943808-54730f72-69eb-4be7-ae73-04dead161841.png)

Reply:
Moved Grid components to functions to resolve the line limit warning.